helm/kvm-operator/ci/default-values.yaml

```diff
@@ -3,6 +3,8 @@ Installation:
   V1:
     Secret:
       Registry:
+        Dockerhub:
+          Token: "token"
```

Comment:
Maybe it is better to have this empty. Then we can check for empty value and fail instead of booting the node and failing to login with wrong credentials

Reply:
This is just for CI linting, no?